gems/aws-sdk-dynamodb/lib/aws-sdk-dynamodb/plugins/simple_attributes.rb

```diff
@@ -78,6 +89,19 @@ module Plugins
       #       }
       #     }
       #
+      # Members are unmarshalled into the following objects:
+      #
+      # * `:m` => `Hash`
+      # * `:l` => `Array`
+      # * `:s` => `String`
+      # * `:n` => `BigDecimal`
+      # * `:b` => `StringIO`
+      # * `:null` => `nil`
+      # * `:bool` => `true` or `false`
+      # * `:ss` => `Set<String>`
+      # * `:ns` => `Set<BigDecimal>`
+      # * `:bs` => `Set<StringIO>`
+      #
```

Comment:
Thank you!

I would move this from the "Input Examples" section to the "Overview". Maybe even right after the first paragraph:

> Simplifies working with Amazon DynamoDB attribute values. Translates attribute values for requests and responses to sensible Ruby natives.